configure.ac

```diff
@@ -393,6 +394,22 @@ if test $ac_cv_libevent_dir != "(system)"; then
   fi
 fi
 
+AC_RUN_IFELSE(
+   [AC_LANG_PROGRAM([
+#include <event.h>
+   ], [dnl
+const char *ver = event_get_version();
+return (ver != NULL && *ver == '2') ? 0 : 1;
```

Comment:
what if libevent ever releases version 3? :)

Reply:
Also, I meant to say that I don't think the daemon even starts with libevent1. There's a runtime check just in case you build against one version but start on another (which is why sanitycheck got added in the first place, I think)

Reply:
yes in fact it s just to make the whole thing compilable :-). if there was no runtime check a compile error pragma would fit better I believe.